cypress/integration/namespaces.spec.js

```diff
@@ -74,12 +74,12 @@ describe('Namespace Select', () => {
 
   it('have the correct namespaces in the dropdown when using navigation header', () => {
     cy.get('@namespace-select-button').click({ wait: 1000 });
-    cy.get('.prose > .text-2xl').contains('Select a namespace');
+    cy.get('.prose > .text-2xl').contains('Select a Namespace');
```

Comment:
Maybe toss a better selector on this element if we are relying on it for a test - although I'm not sure why we are even testing copy 🤔